userspace/libsinsp/parsers.cpp

```diff
@@ -4888,33 +4882,36 @@ void sinsp_parser::parse_fcntl_exit(sinsp_evt *evt)
 
 void sinsp_parser::parse_context_switch(sinsp_evt* evt)
 {
-	if(evt->m_tinfo != nullptr)
+	if(evt->m_tinfo == nullptr)
```

Comment:
For few instances where I felt it was straight forward to do so I changed the logic to "early return" style.